style.css

```diff
@@ -470,3 +482,376 @@ h2 {
   width: 134px;
   border: 5px solid #212121;
 }
+
+@media screen and (min-width: 768px) {
+  .menu {
+    display: none;
+  }
+
+  body {
+    overflow-x: hidden;
+  }
+
+  .links {
+    display: flex;
+    flex-direction: row;
+    list-style-type: none;
+    gap: 24px;
+  }
+
+  .dropdown {
+    display: flex;
+    flex-direction: row;
+    margin-right: 156px;
+    padding-bottom: 30px;
+  }
+
+  a {
+    text-decoration: none;
+    color: #fff;
+    width: 63px;
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .mail {
+    display: block;
+    width: 24px;
+    height: 24px;
+    background-image: url(./Images/Icon\ -\ Mail.png);
+    margin-left: 20px;
+    margin-top: 20px;
+    gap: 30px;
+  }
+
+  header {
+    width: 100%;
+    height: 8px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .headline {
+    width: 100%;
+    height: 550px;
+    background-image: url(/Images/Illustration-header@2x.png);
+    background-size: 500px 768px;
+    background-position: 100% 30%;
+  }
+
+  .brands {
+    display: flex;
+    flex-direction: column;
+    list-style-type: none;
+    width: 20px;
+    height: 98.96px;
+    margin-top: 150px;
+  }
+
+  .icon {
+    display: block;
+    padding: 10px;
+  }
+
+  .headline-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 192px 0;
+    position: absolute;
+    width: 566px;
+    height: 500px;
+    left: 143px;
+    top: 72px;
+  }
+
+  .title {
+    display: block;
+    padding: 0;
+    margin-left: 220px;
+    width: 556px;
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    font-weight: 500;
+    line-height: 44px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: auto;
+  }
+
+  p {
+    font-family: 'Inter', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 32px;
+    letter-spacing: 0;
+    text-align: left;
+    margin-top: 0;
+  }
+
+  .About-myself {
+    width: 100%;
+    height: 600px;
+  }
+
+  .Recent-work {
+    height: 52px;
+    width: 100%;
+    margin-top: 0;
+  }
+
+  .middle {
+    display: flex;
+    flex-direction: row;
+    margin-left: 150px;
+    gap: 24px;
+  }
+
+  .about-img {
+    background-image: url(/Images/img-Placeholder.png);
+    height: 448px;
+    width: 684px;
+    left: 0;
+    top: 0;
+  }
+
+  .rightblock {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 14px;
+    height: 384px;
+    margin-top: 50px;
+  }
+
+  .Multi-Post-ul {
+    height: 40px;
+    width: 297px;
+    left: 0;
+    top: 236px;
+    border-radius: 0;
+  }
+
+  .work-container {
+    height: 448px;
+    width: 100%;
+    border-radius: 0;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    gap: 20px;
+    justify-items: center;
+    align-items: center;
+  }
```

Comment:
- [ ] Good job making most of the changes @arnoldnekemiah, But as mentioned by the previous reviewer, your work section is still not looking like the one in the figma design. Unfortunately, your cards are not visible and we need to fix this. 
### Your work section 
![image](https://user-images.githubusercontent.com/69077061/222876894-49b150ad-22db-4821-bf28-67e0cde8bfe5.png)

Reply:
Hi @TadesseDev  i am really not understanding how these images are not showing yet in my browser they are showing

Reply:
Hello @arnoldnekemiah, Please use some incognito browser tab to see this. You might have some extension running that inverts the color. The problem is the color, the cards are still there. But it's white on white, that is why it's not visible.

Reply:
Hello @TadesseDev even with an incognito browser the images still show.

Reply:
may be the mistake kud be in the naming of the images, let me remove spaces